README.rst

```diff
@@ -10,21 +10,93 @@ but the current implementation does not yet support it.
 .. _Django framework: https://www.djangoproject.com/
 .. _Common Workflow Language: https://github.com/common-workflow-language/common-workflow-language
 
-Install::
+-------
+Install
+-------
+
+.. _install-prerequisites:
+
+Prerequisites
+-------------
+
+Make sure you have Python_ (2.7 or 3.3+) installed on your system. If you don't
+have it yet, follow `these instructions
+<https://docs.python.org/3/using/index.html>`__.
+
+Resolwe requires PostgreSQL_ (9.4+). Many Linux distributions already include
+the required version of PostgreSQL (e.g. Fedora 22+, Debian 8+, Ubuntu 15.04+)
+and you can simply install it via distribution's package manager.
+Otherwise, follow `these instructions
+<https://wiki.postgresql.org/wiki/Detailed_installation_guides>`__.
+
+.. _Python: https://www.python.org/
+.. _PostgreSQL: http://www.postgresql.org/
+
+Installing via pip
+------------------
+
+*Resolwe is not availabe via PyPI yet.*
+
+Installing from source
+----------------------
+
+This installation method will install all Resolwe's dependencies from PyPI_.
+Installing the ``psycopg2`` dependency will require having a C compiler
+(e.g. GCC_) as well as Python and PostgreSQL development files installed on
+the system.
+
+.. note::
+
+    The preffered way to install the C compiler and Python and PostgreSQL
+    development files is to use your distribution's packages, if they exist.
+    For example, on a Fedora/RHEL-based system, that would mean installing
+    ``gcc``, ``python-devel``/``python3-devel`` and ``postgresql-devel``
+    packages.
+
+Download the `latest release of Resolwe
```

Comment:
I think we should keep the wording and change the link when we have the first release.